src/frontends/tests/frontend/shared/src/op_fuzzy.cpp

```diff
@@ -65,10 +65,22 @@ void FrontEndFuzzyOpTest::runConvertedModel(const std::shared_ptr<ngraph::Functi
                                             const std::string& modelFile) {
     auto modelFolder = getModelFolder(modelFile);
 
+    // reshape for models with dynamic input shape
+    const auto parameters = function->get_parameters();
+    for (size_t i = 0; i < parameters.size(); i++) {
+        if (parameters[i]->get_partial_shape().is_dynamic()) {
+            // read input npy file
+            std::string dataFile = modelFolder + "/input" + std::to_string((parameters.size() - 1) - i) + ".npy";
+            cnpy::NpyArray input = cnpy::npy_load(dataFile);
+
+            parameters[i]->set_partial_shape(Shape(input.shape));
+        }
+    }
+    function->validate_nodes_and_infer_types();  // FIXME: in need? any better way to reshape?
+
     // run test
-    auto testCase = test::TestCase(function, "CPU");
+    auto testCase = test::TestCase(function, "TEMPLATE");
```

Comment:
Better to keep 'CPU' to keep speed.

Reply:
There would be problem with multiclass/matrix nms unit tests with CPU plugin as the output are padded to the upper boundary (static output shape). 
@Ilya-Krylov how do you think of this? Switch to TEMPLATE plugin as we discussed before or overload the comparison which should not ask the output shape strictly matching, compatible instead?
https://github.com/openvinotoolkit/openvino/blob/070f27a0892661367ca92f081aa6994d0c3be004/src/tests/engines_util/test_case.cpp#L152

Reply:
@ilya-lavrenov is this message for you?

Reply:
sorry for typo. I corrected.

Reply:
@ceciliapeng2011 I don't remember right why do we have any issues on CPU?
Because at the current moment, CPU also supports dynamic shapes.

If you switch CPU to TEMPLATE we need to understand the performance degradation.

Reply:
done